cfme/tests/configure/test_display_settings.py

```diff
@@ -38,9 +32,5 @@ def test_timezone_setting(appliance, set_timezone):
     Metadata:
         test_flag: visuals
     """
-    locator = ('//label[contains(@class,"control-label") and contains(., "Started On")]'
-               '/../div/p[contains(., "{}")]'.format("-1000"))
-
-    navigate_to(current_appliance.server, 'DiagnosticsDetails')
-
-    assert appliance.browser.widgetastic.is_displayed(locator), "Timezone settings Failed"
+    view = navigate_to(appliance.server, 'DiagnosticsDetails')
+    assert '-1000' in view.summary.started_on.text, 'Timezone settings Failed'
```

Comment:
Consider creating a module tuple of the timezone and offset and referring to it here and above on L24 when setting timezone. DRY, basically.

Reply:
Something like this? Or am I misunderstanding you?
```
test_timezone = ('(GMT-10:00) Hawaii', '-1000')
...
appliance.user.my_settings.visual.timezone = test_timezone[0]
...
assert test_timezone[1] in view.summary.started_on.text, 'Timezone settings Failed'
```

Reply:
Yep, can use namedTuple if you want to use meaningful attribute names instead of index integers.